<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="User-PC" name="AttemptOne2" tests="1" failures="1" timestamp="11 May 2017 03:31:38 GMT" time="6.48" errors="0">
  <testcase name="SurveyData1" time="6.479" classname="Student.AttemptSurvey">
    <failure type="java.lang.AssertionError" message="Launch Applicatin Failed for:http://localhost/ expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Launch Applicatin Failed for:http://localhost/ expected [true] but found [false]
	at Common.LaunchApp.Execute(LaunchApp.java:58)
	at Student.AttemptSurvey.SurveyData1(AttemptSurvey.java:31)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- SurveyData1 -->
</testsuite> <!-- AttemptOne2 -->
